deploy/kcp-hacbs-workspace-init/templates/secret.yaml

```diff
@@ -4,4 +4,11 @@ metadata:
   name: jvm-build-secrets
 stringData:
   registry.token: {{ .Values.quayToken }}
-  .dockerconfigjson: {{ .Values.quayToken }}
+  .dockerconfigjson: |
```

Comment:
given the PR description that this is not a long term thing, maybe put a TODO comment about eventually removing this, and if possible detail when/what triggers the removal

thanks